### ISSUE TITLE
Tirar especificação de abrigos de mulheres e crianças na visualização pública

### ISSUE THREAD

**Issue by giggio** [SOS-RS/frontend#255]
# TL;DR

Por motivos de segurança (e da LGPD), precisamos retirar a descrição “abrigo para mulheres e crianças” do título dos abrigos listados. 

# Critérios de aceitação

We will accept when the shelter's list don't have any titles involving women or children. There are no texts public showing women or children.

There will be no public data specifying shelters for women and children.

Users shouldn't be able to search woman and children.

1. Retirar a informação “Abrigo exclusivo mulheres de crianças” do título de qualquer abrigo que tenha essa descrição

![image](https://github.com/SOS-RS/frontend/assets/334958/18a8c386-82ce-4293-a27b-e9ee6f820812)

**Comment by felipemrvieira** [comment]
Isso vai ser tratado no front?

**Comment by samiraetc** [comment]
Acho que essa informação faz mais sentido de ser tratado no backend

**Comment by diegodario88** [comment]
Talvez seja necessário ajustar a inserção (front-end) de novos abrigos para que essas informações não sejam incluídas. Além disso, no back-end, podemos executar uma migration para atualizar os abrigos já cadastrados cujo título contenha as palavras "exclusivo", "mulheres" ou "crianças".

**Comment by samiraetc** [comment]
@diegodario88 o cadastro de novos abrigos não é feito através do formulário disponibilizado [no site](https://forms.gle/2S7L2gR529Dc8P3T9)?

**Comment by diegodario88** [comment]
@samiraetc sim, você está correto. Talvez fosse interessante adicionar uma observação ao formulário para evitar adicionar informações que possam identificar as pessoas

**Comment by giggio** [comment]
Isso vai ser tratado no frontend e no backend. Vejam: https://github.com/SOS-RS/backend/issues/141

**Comment by WellytonSdJ** [comment]
Olá pessoas! Espero que estejam bem!
Se possivel queria assumir essa tarefa.
